### PR TITLE
Bump toolchain/elemental-cli to 0.20220123

### PR DESCRIPTION
**Pull request by cOS-cibot**
Signed-off-by: cOS-cibot [bot] <cOScibot@gmail.com>

JavaScript Arrays

### CODE REVIEW

File: packages/toolchain/elemental-cli/collection.yaml [hunk 1]
```diff
@@ -3,13 +3,13 @@ packages:
...
     bin_name: "elemental"
-    version: 0.0.13-0
+    version: "0.20220123"
     fips: false
     labels:
       github.repo: "elemental-cli"
       github.owner: "rancher-sandbox"
       autobump.revdeps: "true"
-      git.hash: "d1d11006f556aacaddf82a403894a4445b7b696c"
+      git.hash: "null"
```
Comment (Itxaka) [inline packages/toolchain/elemental-cli/collection.yaml:12]: Wat

**Review by Itxaka — CHANGES_REQUESTED** [review]

File: packages/toolchain/elemental-cli/collection.yaml (hunk 1 above)
Comment (mudler) [reply packages/toolchain/elemental-cli/collection.yaml:12]: maybe  the repo rename ?

Comment (Itxaka) [reply packages/toolchain/elemental-cli/collection.yaml:12]:
```
$ curl https://api.github.com/repos/rancher-sandbox/elemental-cli/git/refs/heads/master
{
  "message": "Moved Permanently",
  "url": "https://api.github.com/repositories/433378128/git/refs/heads/master",
  "documentation_url": "https://docs.github.com/v3/#http-redirects"
}
```

```
$ curl https://api.github.com/repositories/433378128/git/refs/heads/master
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest"
}

```

```
$ curl https://api.github.com/repos/rancher-sandbox/elemental/git/refs/heads/master            
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest"
}
```


:exploding_head:

Comment (Itxaka) [reply packages/toolchain/elemental-cli/collection.yaml:12]: duh

its MAIN not MASTER ¬_¬

**Pull request closed without merging**